tox.ini

```diff
@@ -99,7 +99,7 @@ changedir =
   exporter-zipkin-combined: exporter/opentelemetry-exporter-zipkin/tests
   exporter-zipkin-proto-http: exporter/opentelemetry-exporter-zipkin-proto-http/tests
   exporter-zipkin-json: exporter/opentelemetry-exporter-zipkin-json/tests
-
+  
```

Comment:
This was probably not intended, right? :grimacing:

Reply:
Removed changes in `tox.ini`